### PR TITLE
[OpenBMC] Support obtaining the status of physical LED lights on the Server

### PR DESCRIPTION
**Pull request by whowutwut**
Looking at the physical server and looking at the LED API, I create this pull request to get the status of the LEDs on the P9 OpenBMC Servers 

### Node is powered off, Fault light is on
```
[root@briggs01 ~]# rvitals mid05tor12cn18 leds 
mid05tor12cn18: Front . . : Power:Blink Fault:On Identify:Off
mid05tor12cn18: Front Fans: No LEDs On
mid05tor12cn18: Rear. . . : Power:Blink Fault:On Identify:Off
```

### Node is powered on, fault light is on, fan warning lights are on fan0, fan2, and fan3
```
[root@boston02 ~]# rvitals sn01 leds
sn01: Front . . : Power:On Fault:On Identify:Off
sn01: Front Fans: fan0:On fan1:Off fan2:On fan3:On
sn01: Rear. . . : Power:On Fault:On Identify:Off
```

### Over a range of nodes passed through `xcoll` 

```
[root@briggs01 ~]# rvitals p9up leds | grep -v debug | xcoll
====================================
mid05tor12cn13,mid05tor12cn15,mid05tor12cn02,mid05tor12cn05
====================================
Front . . : Power:On Fault:Off Identify:Off
Front Fans: No LEDs On
Rear. . . : Power:On Fault:Off Identify:Off

====================================
mid05tor12cn16
====================================
Front . . : Power:On Fault:On Identify:Off
Front Fans: No LEDs On
Rear. . . : Power:On Fault:On Identify:Off

====================================
mid05tor12cn18
====================================
Front . . : Power:Blink Fault:On Identify:Off
Front Fans: No LEDs On
Rear. . . : Power:Blink Fault:On Identify:Off
```

### Turn `rbeacon` on and check leds...
```
[root@briggs01 ~]# rbeacon mid05tor12cn15 on
mid05tor12cn15: on
[root@briggs01 ~]# rvitals mid05tor12cn15 leds 
mid05tor12cn15: Front . . : Power:On Fault:Off Identify:Blink
mid05tor12cn15: Front Fans: No LEDs On
mid05tor12cn15: Rear. . . : Power:On Fault:Off Identify:Blink
```

I'm open for suggestions on how to improve the formatting or even making a more condensed format, but I do think this could be potentially useful to quickly prune out the nodes that are unhealthy for some reason.

### CODE REVIEW

**Comment by whowutwut** [comment]
@xuweibj This is ready for review 

New output after last commit: 

```
[root@fs2vm110 ~]# rvitals f6u17 leds
f6u17: Front . . . . . : Power:Blink Fault:Off Identify:Off
f6u17: Front Fans  . . : No LEDs On
f6u17: Rear  . . . . . : Power:Blink Fault:Off Identify:Off


[root@boston02 ~]# rvitals sn01 leds
sn01: Front . . . . . : Power:On Fault:On Identify:Off
sn01: Front Fans  . . : fan0:On fan1:Off fan2:On fan3:On
sn01: Rear  . . . . . : Power:On Fault:On Identify:Off
```

File: xCAT-server/lib/xcat/plugins/openbmc.pm [hunk 1]
```diff
@@ -3605,53 +3618,104 @@ sub rvitals_response {
...
+    my $f1 = "";
+    my $f2 = "";
+    my $f3 = "";
+    my $front_id = "";
+    my $front_fault = "";
+    my $front_power = "";
+    my $rear_id = "";
+    my $rear_fault = "";
+    my $rear_power = "";
+
```
Comment (xuweibj) [inline xCAT-server/lib/xcat/plugins/openbmc.pm]: Hi @whowutwut , why not use hash to save all variables?

**Comment by whowutwut** [comment]
@xuweibj I made some change to use a hash...  

After changes, the results are same: 
```
[root@fs2vm110 ~]# rvitals f6u17 leds
f6u17: Front . . . . . : Power:Blink Fault:Off Identify:Off
f6u17: Front Fans  . . : No LEDs On
f6u17: Rear  . . . . . : Power:Blink Fault:Off Identify:Off
```
 and 
```
[root@boston02 ~]# rvitals sn01 leds
sn01: Front . . . . . : Power:On Fault:On Identify:Off
sn01: Front Fans  . . : fan0:On fan1:Off fan2:On fan3:On
sn01: Rear  . . . . . : Power:On Fault:On Identify:Off
```

**Comment by xuweibj** [comment]
Looks good to me. Thanks @whowutwut

**Pull request merged**